simpeg/regularization/pgi.py

```diff
@@ -70,10 +70,10 @@ class PGIsmallness(Smallness):
         ``tensor``, ``QuadTree`` or ``Octree`` meshes.
     approx_gradient : bool
         If ``True``, use the L2-approximation of the gradient by assuming
-        physical property values of different types are uncorrelated.
```

Comment:
Oh yeah I remember correcting that statement somewhere else (maybe directives).
Disclaimer: I did not write those docstring. But the fault lies with me if there are inaccurate (I should have wrote them in the first place). That is to say, the person who wrote those did to their best of their understanding, but you should feel free to improve it.